arm-postgresql/2017-04-30-preview/swagger/postgresql.json

```diff
@@ -896,8 +896,21 @@
     },
     "ServerPropertiesForCreate": {
       "discriminator": "createMode",
-      "required": [ "createMode" ],
+      "required": [
+        "createMode",
+        "administratorLogin",
+        "administratorLoginPassword"
+      ],
       "properties": {
+        "administratorLogin": {
+          "type": "string",
+          "description": "The administrator's login name of a server. Can only be specified when the server is being created (and is required for creation)."
+        },
+        "administratorLoginPassword": {
+          "type": "string",
+          "format": "password",
+          "description": "The password of the administrator login."
```

Comment:
Thanks for the context, @Suna-MS . It does seem like fixing the generator is the better way. 

@tombuildsstuff : given the discussion around this in #1309 , would you be willing to drop commit 9adaafd and update this PR? we'll keep the change to `create` method but wait on the Go generator to fix the credentials issue. Is that acceptable?